app/views/static_pages/home/_curriculum.html.erb

```diff
@@ -9,14 +9,15 @@
         <div class="row curriculum-details-grid">
 
           <% available_courses.each do |course| %>
-          <div class="curriculum-details-tile align-items-center">
-            <%= image_tag course[:badge_image_url], alt: course[:badge_alt_text] %>
-            <h4 class="bold text-uppercase">
-              <%= course[:title] %>
-            </h4>
-          </div>
+            <div class="curriculum-details-tile align-items-center">
+            <%= link_to course[:path], :class =>"course-tile-path" do %>  
+                <%= image_tag course[:badge_image_url], alt: course[:badge_alt_text] %>
```

Comment:
Indentation looks off in these lines. Please make sure that all the lines you add/change are indented by 2 spaces to keep them consistent with the rest of the code in the app.

Reply:
done :)